components/proxy/src/main/java/com/hotels/styx/admin/AdminServerBuilder.java

```diff
@@ -115,7 +115,9 @@ public HttpServer build() {
         httpRouter.add("/admin/status", new StatusHandler(healthCheckHandler));
         httpRouter.add("/admin/ping", new PingHandler());
         httpRouter.add("/admin/threads", new ThreadsHandler());
-        httpRouter.add("/admin/metrics", new MetricsHandler(environment.metricRegistry(), metricsCacheExpiration));
+        MetricsHandler metricsHandler = new MetricsHandler(environment.metricRegistry(), metricsCacheExpiration);
+        httpRouter.add("/admin/metrics", metricsHandler);
+        httpRouter.add("/admin/metrics/", metricsHandler);
         httpRouter.add("/admin/healthcheck", healthCheckHandler);
         httpRouter.add("/admin/configuration", new StyxConfigurationHandler(staticConfiguration()));
         httpRouter.add("/admin/configuration/origins", new OriginsHandler(backendServicesRegistry));
```

Comment:
This feature needs to be described in the Styx user manual. Please add relevant information there.